examples/hlb_cifar10.py

```diff
@@ -308,4 +356,17 @@ def eval_step_jitted(model, X, Y):
     i += 1
 
 if __name__ == "__main__":
-  train_cifar()
+  if not getenv("DIST"):
+    train_cifar()
+  else: # distributed
+    devices = ["gpu:0", "gpu:1", "gpu:2", "gpu:3", "gpu:4", "gpu:5"]
```

Comment:
we don't have a good way of detecting how many gpus are in the system, because stuff like opencl can return cpu runtimes which we don't want to mix in. 

There is other stuff as well, like you have to ensure that your `BS` and `EVAL_BS` are divisible by your world size.

Reply:
is the issue that you have to init in order to know? like the CPU thing is dealt with fine in the gpu code now

Reply:
i think we can just call `cl.get_platforms` and it shouldn't break anything because i don't think that initializes the gpus.